x/incentives/keeper/suite_test.go

```diff
@@ -125,10 +133,14 @@ func (suite *KeeperTestSuite) setupNewGaugeWithDuration(isPerpetual bool, coins
 }
 
 // TODO: Delete all usages of this method
+// TODO: Why delete all usages of this method?
+
+// Creates a gauge with the default lock duration.
 func (suite *KeeperTestSuite) SetupNewGauge(isPerpetual bool, coins sdk.Coins) (uint64, *types.Gauge, sdk.Coins, time.Time) {
```

Comment:
TODO: Why delete all usages of this method?

Reply:
Curious about this as well